traffic_router/core/src/test/resources/publish/CrStates

```diff
@@ -146,6 +149,30 @@
       "disabledLocations": [],
       "isAvailable": true
     },
+    "client-steering-diversity-test": {
```

Comment:
Not sure if it would be of value to test CS diversity with a case where disabled locations for a target is populated.

Reply:
I think the `disabledLocations` list is mostly just an optimization (an unnecessary one IMO but that's a different issue) for Traffic Monitor to tell Traffic Router that entire cachegroups are currently unavailable for a delivery service so that TR can eliminate those cachegroups from its "cache selection process". The CSD stuff isn't really changing that cache selection process since CSD is operating on the _result_ of that process to diversify the end result. So, I'm thinking that test case would add more coverage around the "cache selection process" but not really CSD per se.